packages/fxa-settings/src/lib/gql.test.ts

```diff
@@ -74,4 +77,26 @@ describe('errorHandler', () => {
 
     expect(window.location.replace).not.toHaveBeenCalled();
   });
+
+  it('does not redirect if the current page does not require the user to be authenticated', () => {
+    delete window.location;
+    window.location = {
+      replace: jest.fn(),
+      search: '',
+      pathname: 'foo',
+      href: pageWhichDoesNotRequireAuthentication,
+    };
+    let networkError: any;
```

Comment:
nit: Should this be `any` or `Error` or something in between?
`any` feels pretty loose.